src/main/java/org/broadinstitute/hellbender/utils/test/IntegerReadCountFileComparator.java

```diff
@@ -0,0 +1,112 @@
+package org.broadinstitute.hellbender.utils.test;
+
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.copynumber.coverage.readcount.ReadCountFileHeaderKey;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.testng.Assert;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+/**
+ * This is a testing utility class to check the correctness of coverage collection results for CNV analysis.
+ * It assumes that results come in a TSV format, and checks for exact string matches between two tables. Therefore,
+ * this class is NOT intended to be used with output formats containing floats, as it doesn't allow for rounding error.
+ */
+public class IntegerReadCountFileComparator {
```

Comment:
I don't know about calling this a Comparator, since it doesn't impose an ordering?